packages/components/molecules/f-media-element/src/components/MediaElement.vue

```diff
@@ -0,0 +1,35 @@
+<template>
+    <div
+        :class="$style['c-mediaElement']"
+        data-test-id="mediaElement-component">
+        mediaElement
+    </div>
+</template>
+
+<script>
+
+export default {
+    name: 'MediaElement',
```

Comment:
Don't need to set name inside components, vue-cli adds this automatically.

Reply:
This is generated output, could you please add this to the generator PR as a comment thanks, will make the change :)